osCam/pathbuilder/views.py

```diff
@@ -0,0 +1,284 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+def getNextPathList(curUser: User, pathBuilder: Path) -> list:
+	# returns alist of children paths relative to our current Path in FileTree
+	nextPaths = []
+	print("Get Next Path List ??")
+	for _name in os.listdir(pathBuilder.path):
+		subdir = os.path.join(pathBuilder.path, _name)
+		if os.path.isdir(subdir):
+			print(_name)
+			nextPaths.append(_name)
+
+	return nextPaths	
+
+def updatePathNext(pid: int, nextPaths: list) -> None:
+	# Updates a Path's 'NEXT" entry, such that a path has a LIST of its children $paths
+	print("Updating Path.next [it] {}".format(str(pid)))
+	_curPath = Path.objects.get(id=pid)
+	# it = _curPath.next
+	print("Looking at [it] {}".format(str(_curPath.path)))
+	for _next in nextPaths:
+		# it:NextPath = NextPath.objects.create(
+		# 	data=_next
+		# )
+		new_next = _curPath.next_set.create(
+			data=_next
+		)
+		print("FOUND Data: ",new_next.data)
+		print("Register: {} -- {}".format(_curPath.path,new_next.data))
+		# _curPath.
+		# _curPath.save()
+
+
+def getFilePathContext(request) -> dict:
+	# path = []
+	# cur_path = None
+	builder = request.POST.get("$PATH")
+	print("Builder: {}".format(builder))
+	# current_path
+	this_user = User.objects.get(id=request.user.id)
+	nextPaths:list = []
+	# default_path:Path = None
+	default_path_builder: Path = Path.objects.get(user=this_user)
+	# print(f"Most Recent Path: {default_path.path} \n\n")
+	nextPaths = getNextPathList(curUser=this_user, pathBuilder=default_path_builder)
+	context = {"pathBuilder":default_path_builder, "nextPathList": nextPaths}
+	
+	# return (cur_path,nextPaths)
+	return context
+	
+def displaySetStoragePath(request):
+
+	# 	# =(1) get All 'next' paths set up in DB.model
+	# 	# (2) Set Storage mode to UPDATE
+	# 	# (3) append new Path
+	# pathDataDialog = PathDataDialog()
+	this_user = User.objects.get(id=request.user.id)
+	next_path_list = []
+	_full_path=Path.objects.filter(user=this_user)
+	print("FULL_PATH {}".format(_full_path))
+	full_path=""
+	# for data in _full_path:
+	# 	full_path+=data.path 
+
+	full_path += "/"+request.POST.get("$PATH")
+	cur_path_builder: Path = None 
+	# if no Path objects exist yet for this user
+	# - we create one and peek at its children
+	if Path.objects.filter(user=this_user).exists():
+	# 	# turn on Storate update permissions.
+		cur_path_builder = Path( 
+			user=this_user,
+			path= "/" + request.POST.get("$PATH"),
+		)
+		
+	# 	next_path_list=getNextPathList(this_user, this_path.path)
```

Comment:
lots of commented out code, please remove

Reply:
@melippmann  the code you mentioned is removed and the PR is updated for your review. Note, if above the comment says "outdated" like this one does (in orange) it's usually because the person who pushed the PR has updated the page.. Also you should not have to see this pathBuilder page along with a couple others that accidently were in the bundle. They weren't ready.

Reply:
This file is not listed as "outdated" for me, and I don't see the changes reflected.

I thought that if you commit a change while the PR is still open that the change should then be reflected in the files for review?